### PR TITLE
Added Favourites on selected recipes

### PR DESCRIPTION
**Pull request by AkuaQ**
**Added Favourites on selected recipes**
Recipes are now saved to a favourites database with the click of a button.

Next: 
- Displaying favourites from database in objective C (figuring out so prints will leave when sorted)
- Refactoring to MVVM of favourites feature

### CODE REVIEW

File: FirstApp/FavouritesObjectiveCViewController.m [hunk 1]
```diff
@@ -18,6 +18,7 @@ @implementation FavouritesObjectiveCViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     [self.navigationController.navigationBar.topItem setTitle:@"Food Finder"];
+    NSLog(@"Favourites: %@", _username);
```
Comment (Tiewhan) [inline FirstApp/FavouritesObjectiveCViewController.m]: NS Logs shouldn't appear when being merged. Clutter and wastes resources

File: FirstApp/View/HomeViewController.swift [hunk 2]
```diff
@@ -33,6 +33,7 @@ class HomeViewController: UIViewController {
         }
         recipeSearchBar.delegate = self
         self.hideKeyboardWhenTappedAround()
+        print("Home: \(username)")
```
Comment (Tiewhan) [inline FirstApp/View/HomeViewController.swift]: Same issue as with Log statements, shouldn't be here

File: FirstApp/View/RecipeDetailViewController.swift [hunk 3]
```diff
@@ -35,6 +39,44 @@ class RecipeDetailViewController: UIViewController {
...
+            database.collection("favourites").whereField("username", isEqualTo: username).whereField("title",
+                                                                                                     isEqualTo: recipeTitle)
+                .getDocuments() { (snapshot, err)
+                    in
+                if let err = err {
+                    print("Error getting documents: \(err)")
+                } else {
+                    snapshot?.documents.first?.reference.delete()
+                }
+            }
```
Comment (Tiewhan) [inline FirstApp/View/RecipeDetailViewController.swift]: I would advice that something like this be in its own function. Makes the code much more readable and if you need the code again you have it ready to use.

File: FirstApp/View/HomeViewController.swift (hunk 2 above)
Comment (AkuaQ) [reply FirstApp/View/HomeViewController.swift]: Do you read my pull request message😅

File: FirstApp/View/RecipeDetailViewController.swift (hunk 3 above)
Comment (AkuaQ) [reply FirstApp/View/RecipeDetailViewController.swift]: Cool, will do🙂👍

**Pull request merged**